locations/spiders/tollens_fr.py

```
@@ -0,0 +1,37 @@
+
+TOLLENS = {"brand": "Tollens", "brand_wikidata": "Q127515008"}
+ZOLPAN = {"brand": "Zolpan", "brand_wikidata": "Q126875048"}
+
+
+class TollensFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "tollens_fr"
+    # tollens.com lists the whole Cromology retail network: Tollens and Zolpan
+    # branded stores, plus some co-branded "Tollens Zolpan" ones.
+    item_attributes = TOLLENS
+    sitemap_urls = ["https://www.tollens.com/sitemap.xml"]
+    sitemap_rules = [(r"/nos-magasins/[^/]+/[^/]+$", "parse_sd")]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)